### PR TITLE
Update to the Beta 4 release

### PR DESCRIPTION
**Pull request by sharwell**
:memo: Where possible, I made methods static to avoid having to reorder members to fix SA1204 violations.

### CODE REVIEW

File: StyleCop.Analyzers/StyleCop.Analyzers.Test/DocumentationRules/SA1600UnitTests.cs [hunk 1]
```diff
@@ -500,468 +496,472 @@ public class OuterClass
...
+
+            await this.TestNestedTypeDeclarationDocumentationAsync(type, string.Empty, false, true).ConfigureAwait(false);
+            await this.TestNestedTypeDeclarationDocumentationAsync(type, "private", false, true).ConfigureAwait(false);
+            await this.TestNestedTypeDeclarationDocumentationAsync(type, "protected", false, true).ConfigureAwait(false);
+            await this.TestNestedTypeDeclarationDocumentationAsync(type, "internal", false, true).ConfigureAwait(false);
+            await this.TestNestedTypeDeclarationDocumentationAsync(type, "protected internal", false, true).ConfigureAwait(false);
+            await this.TestNestedTypeDeclarationDocumentationAsync(type, "public", false, true).ConfigureAwait(false);
         }
     }
 }
```
Comment (Noryoko) [inline StyleCop.Analyzers/StyleCop.Analyzers.Test/DocumentationRules/SA1600UnitTests.cs]: Should there be a new line at the end?

File: StyleCop.Analyzers/StyleCop.Analyzers/DocumentationRules/StandardTextDiagnosticBase.cs [hunk 2]
```diff
@@ -122,7 +122,7 @@ private bool SeeTagIsCorrect(SyntaxNodeAnalysisContext context, XmlEmptyElementS
             return actualSymbol.OriginalDefinition == expectedSymbol;
         }
 
-        private string GetName(TypeSyntax name)
+        private static string GetName(TypeSyntax name)
```
Comment (Noryoko) [inline StyleCop.Analyzers/StyleCop.Analyzers/DocumentationRules/StandardTextDiagnosticBase.cs]: Is this used?

Comment (sharwell) [inline StyleCop.Analyzers/StyleCop.Analyzers/DocumentationRules/StandardTextDiagnosticBase.cs]: Apparently not.

File: StyleCop.Analyzers/StyleCop.Analyzers.Test/DocumentationRules/SA1600UnitTests.cs (hunk 1 above)
Comment (sharwell) [inline StyleCop.Analyzers/StyleCop.Analyzers.Test/DocumentationRules/SA1600UnitTests.cs]: There should be.

**Comment by Noryoko** [comment]
:+1:

**Pull request merged**